lib/storage/src/types.rs

```diff
@@ -39,6 +39,11 @@ pub struct PerformanceConfig {
     /// If positive - use this absolute number of CPUs.
     #[serde(default)]
     pub optimizer_cpu_budget: isize,
+    /// IO budget, how many parallel IO operations to allow for an optimization job.
```

Comment:
This might need updating since we don't take just one permit/token for each job anymore.